edx_rest_framework_extensions/tests/test_permissions.py

```diff
@@ -22,15 +23,13 @@ class IsSuperuserTests(TestCase):
 
     @ddt.data(True, False)
     def test_has_permission(self, has_permission):
```

Comment:
Rename to `test_superuser_has_permission`.

Reply:
done.